mujinvisioncontrollerclient/visioncontrollerclient.py

```diff
@@ -20,24 +22,24 @@ class VisionControllerClient(object):
     """Mujin Vision Controller client for binpicking tasks.
     """
 
-    _ctx = None  # type: zmq.Context # zeromq context to use
-    _ctxown = None  # type: zmq.Context
+    _ctx = None  # type: Optional[zmq.Context] # zeromq context to use
+    _ctxown = None  # type: Optional[zmq.Context]
     # if owning the zeromq context, need to destroy it once done, so this value is set
-    hostname = None  # type: str # hostname of vision controller
-    commandport = None  # type: int # command port of vision controller
-    configurationport = None  # type: int # configuration port of vision controller, usually command port + 2
-    statusport = None  # type: int # status publishing port of vision manager, usually command port + 3
+    hostname = None  # type: Optional[str] # hostname of vision controller
+    commandport = None  # type: Optional[int] # command port of vision controller
+    configurationport = None  # type: Optional[int] # configuration port of vision controller, usually command port + 2
+    statusport = None  # type: Optional[int] # status publishing port of vision manager, usually command port + 3
 
-    _commandsocket = None  # type: typing.Optional[zmqclient.ZmqClient]
-    _configurationsocket = None  # type: typing.Optional[zmqclient.ZmqClient]
+    _commandsocket = None  # type: Optional[zmqclient.ZmqClient]
+    _configurationsocket = None  # type: Optional[zmqclient.ZmqClient]
     
     _callerid = None # the callerid to send to vision
     _checkpreemptfn = None # called periodically when in a loop
     
     _subscriber = None # an instance of ZmqSubscriber, used for subscribing to the state
     
     def __init__(self, hostname='127.0.0.1', commandport=7004, ctx=None, checkpreemptfn=None, reconnectionTimeout=40, callerid=None):
-        # type: (str, int, typing.Optional[zmq.Context]) -> None
+        # type: (str, int, Optional[zmq.Context], Optional[Callable], int, Optional[str]) -> None
```

Comment:
Below in the docstring:
```
reconnectionTimeout (float, optional): Sets the "timeout" parameter of the ZmqSocketPool instance
```

is it float or int?

Reply:
float, fixed